Content.Shared/Lathe/LatheMessages.cs

```diff
@@ -9,11 +9,11 @@ public sealed class LatheUpdateState : BoundUserInterfaceState
 {
     public List<ProtoId<LatheRecipePrototype>> Recipes;
 
-    public List<LatheRecipePrototype> Queue;
+    public Queue<ProtoId<LatheRecipePrototype>> Queue;
```

Comment:
`Queue<T>` should not be sent over the network. Use `NetListAsArray<T>` instead.